RELEASE/scripts/autoscend/iotms/mr2024.ash

```diff
@@ -167,3 +167,114 @@ skill dartSkill()
 	}
 	return to_skill(7513); // If there aren't any darts available return the Darts: Throw at %PART1
 }
+
+boolean auto_haveMayamCalendar()
+{
+	if(auto_is_valid($item[Mayam Calendar]) && available_amount($item[Mayam Calendar]) > 0 )
+	{
+		return true;
+	}
+	return false;
+}
+
+boolean auto_MayamIsUsed(string glyph)
+{
+	string[int] used = split_string(get_property("_mayamSymbolsUsed"),",");
+	foreach idx,str in used
+	{
+		if (glyph==str)
+		{
+			return true;
+		}
+	}
+	return false;
+}
+
+boolean auto_MayamClaimStinkBomb()
+{
+	if(!auto_haveMayamCalendar())
+	{
+		return false;
+	}
+	if(auto_MayamIsUsed("vessel") ||
+	   auto_MayamIsUsed("yam2")   ||
+	   auto_MayamIsUsed("cheese") ||
+	   auto_MayamIsUsed("explosion") )
+	{
+		return false;
+	}
+	cli_execute("mayam rings vessel yam cheese explosion");
+	return true;
+}
+
+boolean auto_MayamClaimBelt()
+{
+	if(!auto_haveMayamCalendar())
+	{
+		return false;
+	}
+	if(auto_MayamIsUsed("yam1") ||
```

Comment:
were you getting 2 belts per day? Start of day get belt, temple reset, another belt. Haven't used it so dunno if temple resets resonances as well